jdaviz/configs/default/plugins/virtual_observatory/vo_plugin.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <j-tray-plugin
+    description='Download a data product from the Virtual Observatory'
+    :link="'https://www.ivoa.net/astronomers/index.html'"
+    :popout_button="popout_button">
+    <v-form v-model="all_fields_filled">
+
+      <j-plugin-section-header>Source Selection</j-plugin-section-header>
+
+      <plugin-viewer-select
+        :items="viewer_items"
+        :selected.sync="viewer_selected"
+        label="Viewer"
+        :show_if_single_entry="true"
+        hint="Select a viewer to retrieve center coordinates, or Manual for manual coordinate entry."
+      />
+
+      <v-row v-if="viewer_selected !== 'Manual'">
+        <v-switch
+          v-model="coord_follow_viewer_pan"
+          label="Follow Viewer Center"
+          hint="Automatically adjust coordinates as viewer pans and zooms"
+          persistent-hint
+        ></v-switch>
+      </v-row>
+
+      <v-row>
+        <div :style="!(viewer_selected !== 'Manual' && !coord_follow_viewer_pan) ? 'width: 100%' : 'width: calc(100% - 32px)'">
+          <v-text-field
+            v-model="source"
+            label="Source/Coordinates"
+            hint="Enter a source name or coordinates in degrees to center your query on"
+            :disabled="viewer_selected !== 'Manual'"
+            :rules="[() => !!source || 'This field is required']"
+            persistent-hint>
+          </v-text-field>
+        </div>
+        <div v-if="viewer_selected !== 'Manual' && !coord_follow_viewer_pan" style="line-height:64px; width:32px">
+          <v-btn
+            id="autocenterbtn"
+            @click="center_on_data"
+            :disabled="viewer_centered"
+            icon>
+            <v-icon>
+              {{ viewer_centered ? 'mdi-crosshairs-gps' : 'mdi-crosshairs' }}
+            </v-icon>
+          </v-btn>
```

Comment:
without a label, this would benefit from a tooltip

Reply:
Done in https://github.com/spacetelescope/jdaviz/pull/2872/commits/f6bcdc5cca964e42c62fb8362cece78a00354f9f